api/src/auth/constants.ts

```diff
@@ -0,0 +1,9 @@
+require('dotenv').config();
+
+export const jwtConstants = {
```

Comment:
Huhu, I don't want to be unpleasant. But this time, it's more conventinal to use UpperCase :)

Reply:
done ;)